status/flowcell.py

```diff
@@ -215,19 +214,12 @@ class ONTReportHandler(SafeHandler):
     def __init__(self, application, request, **kwargs):
         super(SafeHandler, self).__init__(application, request, **kwargs)
 
-    def fetch_ont_report(self, name):
-        """ Fetches the MinKNOW .html run report, saved as an escaped string in the
-        database run entry and returns the unescaped string which can be written as .html
-        """
-
-        view_report = self.application.nanopore_runs_db.view("info/minknow_report", descending=True)
-        row = [row for row in view_report.rows if name in row.key][0]
-        str_html = html.unescape(row.value)
-
-        return str_html
-
     def get(self, name):
-        self.write(self.fetch_ont_report(name))
+        
+        reports_dir = "/var/local/minknow_reports"
```

Comment:
This should be in the settings file

Reply:
Good point, fixed in [271dc82](https://github.com/SciLifeLab/genomics-status/pull/728/commits/271dc82be3805826d630ae23464d332a2977e6e6). 

@aanil please update the settings file on ngi-internal accordingly, see [here](https://ngi-git.scilifelab.se/NGI/conf/pulls/72).